tf2_ros/CMakeLists.txt

```diff
@@ -53,12 +58,21 @@ target_link_libraries(buffer_server
   ${PROJECT_NAME}
 )
 ament_target_dependencies(buffer_server
+  ${dependencies}
+)
+
+add_library(static_transform_broadcaster_node SHARED
```

Comment:
Why should the order of the targets be changed?

Reply:
Because the `static_transform_broadcaster` executable depends on the `static_transform_broadcaster_node`.  Thus it seems to make more sense to have the executable after the library it depends on.

Reply:
Sounds good - just makes the diff harder to read.